docs/user-guide/doc-odm-user-guide/studybrowser.md

```diff
@@ -1,84 +1,112 @@
+* You can search by study name, accession number, or any relevant metadata field. 
+* The facets beneath the search bar allow you to filter search results based on sample data (metadata) fields.
 
-The filter panel allows you to filter your results with search facets. Set facets to refine search results. Facets are
-automatically generated from the metadata of available datasets, and the suggested list of facets varies
-depending on a search results. Click ‘More’ to explore the whole list of suggested items.
+![10DataClass.png](doc-odm-user-guide/images/10DataClass.png)
+<figcaption>Filter data. Use the filters to narrow the search of studies. For example, select <strong>Data class</strong> and then tick the option <strong>Expression</strong> to find exclusively studies containing gene expression as experimental data (GCT format files)</figcaption>
 
-![image](doc-odm-user-guide/images/studybrowser_searchpanel.png)
+## Bookmark studies
 
-Bookmarked studies can be shown by clicking on the Bookmarks icon. Access allows you to see your studies, those that are accessible to you (for example, public studies) or those that have been shared with your user account/group.
+* You can save preferred studies for easy access. To the left of a study title is a three-dot link. Click this to open a menu that allows you to share the study (if you are the owner) and save it to bookmarks.
 
-If there are additional facet terms a **Find more** link will be displayed.
-Clicking on this allows you to type in terms and you can select from the presented list of options.
+![11Bookmarks.png](doc-odm-user-guide/images/11Bookmarks.png)
+<figcaption>Bookmark studies. Save preferred studies to access them easily. To do so, click on the three-dot link next to the name of the study and click on <strong>Save to bookmark</strong></figcaption>
 
-![image](doc-odm-user-guide/images/studybrowser_autocomplete.png)
+* Bookmarked studies are accessible by clicking on the Bookmarks icon. This section allows you to see your studies, those accessible to you (e.g., public studies), or those shared with your user account/group.
 
-### Configuring the filter panel
+![12Bookmarked2.png](doc-odm-user-guide/images/12Bookmarked2.png)
+<figcaption>Bookmarked studies. Bookmarked studies are accessible by clicking on the button <strong>Saved</strong>. A new window will appear where you can access the saved studies or remove the bookmark if needed</figcaption>
 
-Exactly which metadata fields are available as search facets can be configured by users who have the permission to configure facets. To change the facets click the cog icon at the bottom of the filter panel.
+## Configuring the filter panel
 
-![image](doc-odm-user-guide/images/configure-facets.png)![image](doc-odm-user-guide/images/configure-facets-screen.png)
+Users with permission (Admins) can customize which metadata fields are available as search facets.
```

Comment:
Let's mention what permission is required to be able to configure filter panel? because user should have certain permission for this, and it's not "Manage organisation" permission

Reply:
Maybe it's better to mention there this part, and in general do it just once: https://github.com/genestack/user-docs/pull/62/files#diff-dbd056400b7234b8002e0708bd85a80ff2bcac83fad9ffefca835db17240b9a2R96

Reply:
I didn't get the second comment. Do you mean to add the description of Configure facets to the Navigation section?